ckan/Dockerfile.dev

```diff
@@ -0,0 +1,38 @@
+FROM openknowledge/ckan-dev:2.8
```

Comment:
Please use ckan-cloud-docker official images.

Reply:
(relevant to many files here)

Reply:
I need a CKAN instance with ckanext-harvester and ckanext-datajson
Should I use viderum/ckan:latest and add the extensions manually?
Should I use viderum/ckan-cloud-docker:ckan-latest-datagov-theme?
I think that with OKF container is enough and is already working.

Reply:
viderum/ckan-cloud-docker:ckan-latest-datagov-theme already has the extenstions, so it would be simplest to use that.

Reply:
It's not simple. I get an error: `/ckan-entrypoint.sh: line 3: source: filename argument required`
We need _$CKAN_K8S_SECRETS_ here.
The OKF version is already working it seems the simpler way now.

Reply:
Maybe simpler but definitely incorrect.
We need to test in a similar environment, against the code that will be running in production.
If you're unsure how exactly to use this image in docker compose, take a look at the docker compose file in ckan-cloud-docker for a working example.

Reply:
It's ready with Viderum containers. Removed all builds